src/commands/moderation/createrole.js

```diff
@@ -0,0 +1,33 @@
+const { Command, SwitchbladeEmbed, Constants } = require('../../')
+
+module.exports = class CreateRole extends Command {
+  constructor (client) {
+    super(client, {
+      name: 'createrole',
+      category: 'moderation',
+      requirements: { guildOnly: true, botPermissions: ['MANAGE_ROLES'], permissions: ['MANAGE_ROLES'] },
+      parameters: [
+        { type: 'color', missingError: 'commands:createrole.noParams' },
+        { type: 'string', full: true, missingError: 'commands:createrole.noParams' }
+      ]
+
+    })
+  }
+
+  async run ({ channel, guild, author, t }, color, name) {
+    const hexcode = color.rgb(true)
+    const embed = new SwitchbladeEmbed(author)
+    await guild.createRole({ name: name, color: hexcode }).then(role => {
+      embed
+        .setTitle(t('commands:createrole.successTitle'))
+        .setDescription(`Name of the role: ${name}`)
```

Comment:
translate this please

Reply:
if having trouble with variables, you can use:
on code `t('string', name)`
and on locale `"string":"Name: {{name}}"`

Reply:
Done!